CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.15.2 (2020-02-25)
+
+- Updated the Webpack configuration handle using `skyux serve` and `skyux test` with IE 11. [#221](https://github.com/blackbaud/skyux-sdk-builder/pull/221)
```

Comment:
Are we saying that the webpack configuration can now use skyux serve and skyux test in IE 11?

Currently, it kind of reads as if `skyux serve` and `skyux test` updated the Webpack configuration.